dotnet/src/webdriver/RelativeBy.cs

```diff
@@ -288,7 +308,7 @@ private RelativeBy Near(object locator, int atMostDistanceInPixels)
 
             Dictionary<string, object> filter = new Dictionary<string, object>();
             filter["kind"] = "near";
-            filter["args"] = new List<object>() { GetSerializableObject(locator), "distance", atMostDistanceInPixels };
+            filter["args"] = new List<object>() { GetSerializableObject(locator), atMostDistanceInPixels };
```

Comment:
I'm just copying what I see in the Java version, and it's beginning to work

https://github.com/SeleniumHQ/selenium/blob/trunk/java/src/org/openqa/selenium/support/locators/RelativeLocator.java#L185

Reply:
All right.